hellsemble/hellsemble.py

```diff
@@ -210,6 +213,13 @@ def __fit_estimators_sequential(
                 X_fit[failed_observations_mask],
                 y_fit[failed_observations_mask],
             )
+
+            # Validate the ensemble
+            val_score = self.evaluate_hellsemble(X_val, y_val)
+            print(f'Validation score for sequential emsemble: {val_score}')
```

Comment:
big "no" for `print` in classes; if it's important, we can:
* store training history in `self.meta` object or
* return it as an additional element

Reply:
The print should be removed. But some logger in DEBUG level should be ok